pages.ko/common/bash.md

```diff
@@ -1,32 +1,33 @@
 # bash
 
-> Bourne-Again SHell. `sh`-호환 명령 행 인터프리터.
-> 더 많은 정보: <https://gnu.org/software/bash/>.
+> Bourne-Again SHell, an `sh`- 호환 명령 행 인터프리터.
+> 참조 : `zsh`, `histexpand` (history expansion).
+> 더 많은 정보는 여기서 찾아볼 수 있습니다: <https://gnu.org/software/bash/>.
 
-- 대화식 쉘 시작:
+- 대화형 쉘 시작하기: 
 
 `bash`
 
-- 명령 실행:
+- 설정 파일 로딩 없이 대화형 쉘 시작하기:
 
-`bash -c "{{command}}"`
+`bash --norc`
 
-- 파일에서 명령 실행:
+- 특정 명령어([c]ommands) 실행하기:
 
-`bash {{file.sh}}`
+`bash -c "{{echo 'bash is executed'}}"`
 
-- 파일에서 명령 실행하고, 터미널에서 실행 된 모든 명령 기록:
+- 특정 스크립트 실행하기:
 
-`bash -x {{file.sh}}`
+`bash {{path/to/script.sh}}`
 
-- 파일에서 명령 실행하고, 첫 번째 에러에서 중지:
+- 각 명령어 실행 전 명령어 인쇄하며 스크립트 실행하기:
 
-`bash -e {{file.sh}}`
+`bash -x {{path/to/script.sh}}`
 
-- stdin에서 명령 실행:
+- 에러([e]rror)가 발생하면 멈추도록 특정 스크립트 실행하기:
 
-`bash -s`
+`bash -e {{path/to/script.sh}}`
```

Comment:
Can you translate the path too?

Reply:
@kbdharun Thanks for the swift feedback! 
you mean like "execute a `path/to/script.sh` script ~~ " ?

Reply:
yeah, `path/to/script.sh`

Reply:
got it!